phockup.py

```diff
@@ -1,409 +1,4 @@
 #!/usr/bin/env python3
-import argparse
-import logging
-import logging.handlers
-import os
-import re
-import sys
+from phockup import app
 
-from src.date import Date
-from src.dependency import check_dependencies
-from src.phockup import Phockup
```

Comment:
Seems that this workaround work. If it's really working, we probably may have backward compatibility, but I'm not really sure.